subprojects/core-api/src/main/java/org/gradle/api/Task.java

```diff
@@ -841,9 +841,13 @@ public String determineName(Task task) {
     /**
      * Registers a {@link BuildService} that is used by this task so
      * {@link BuildServiceRegistration#getMaxParallelUsages() its constraint on parallel execution} can be honored.
+     * <p>
+     * This is not necessary for task properties declared as {@link org.gradle.api.services.ServiceReference}s.
+     * </p>
```

Comment:
💅  We are not using closing paragraph tags in javadoc as per our javadoc style guide

Reply:
It is quite widespread on this file though... I kinda feel local consistency is more important.

Reply:
Will drop the closing tag I introduced in the other source files